README.md

```diff
@@ -1,11 +1,23 @@
-# Subspace Network Monorepo
+<h1 align="center">
```

Comment:
Honestly I don't think this huge header is necessary. The way it was before was perfectly fine and build icons were working, now they are broken. HTML should be reduced, ideally to zero. Linking resources from website that will change later anyway is not good. Also it looks ugly with dark mode on GitHub.

I'd just add 2 more badges and leave the rest as it was.